src/main/java/io/github/moulberry/notenoughupdates/util/SBInfo.java

```diff
@@ -294,6 +299,14 @@ public void tick() {
 			lastLocRaw = System.currentTimeMillis();
 			NotEnoughUpdates.INSTANCE.sendChatMessage("/locraw");
 		}
+		try {
+		if (currentTime - lastMayorUpdate > 300 * 1000) {
+			mayorJson = Utils.getCurrentMayor();
+			lastMayorUpdate = currentTime;
+		}
+			} catch (ExecutionException | InterruptedException e) {
+				throw new RuntimeException(e);
```

Comment:
Crashing the users game when something goes wrong with the request does not seem like a good idea